actions/setup-conftest/action.yaml

```
@@ -0,0 +1,51 @@
+name: Install conftest
+description: Install conftest
+
+inputs:
+  version:
+    description: |
+      Version of conftest to install.
+    default: 0.55.0
+
+runs:
+  using: composite
+
+  steps:
+    - name: Restore cache
+      id: restore
+      uses: actions/cache/restore@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+      with:
+        path: ${{ github.workspace }}/bin
```

optional: just in case the directory happens to have things in it already e.g. prepopulated on the runner or put there by a previous step?

```suggestion
        path: ${{ github.workspace }}/bin/conftest
```